### DISCUSSION TITLE
Mukurtu 4 New Features (Post-Launch)

### DISCUSSION THREAD

**Discussion by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#13] [New Feature Request]
This is a non-exhaustive list of new and updated features, functions, and tools that are under consideration or exploration for inclusion in future Mukurtu releases, AFTER the initial launch is complete. There is no set timeframe for these items, but a tentative roadmap is available [here](https://mukurtu.org/mukurtu-4/).

- Improved user management
- Improved commenting system
- Improve translation and localization tools
- OHMS (Oral History Metadata Synchronizer) integration
- Additional metadata schema support
- Introducing standard Drupal 8/9 theming more readily
- 3D media in DH items
- Enhanced Mapping functionality
- Expanded Taxonomy records (eg: person records, place records)

**Comment by kenianbei** [comment]
I'm curious if you plan to use the core media module for media management for D8/9? One problem we are having with D7 is that there is no easy way to remotely store media files. For example something like the AWS S3 module: https://www.drupal.org/project/s3fs.

**Reply by taylor-steve** [reply]
Yes, we are currently using the core Drupal 8/9 media entities to replace Scald from Mukurtu v2/3. I haven't used the s3fs module lately but I believe if it was configured to replace the 'private://' URI file handling everything would work. We're not currently testing that configuration for Mukurtu v4, but we'll add it to the list to examine further. That's something I think we'd like to use internally as well, so chances are good that we'll make it work.

Also it looks like Michael has just now created a new discussion topic specifically for media [here](https://github.com/MukurtuCMS/Mukurtu-CMS/discussions/14) if you've got any additional thoughts/questions regarding media. Thanks!

**Reply by kenianbei** [reply]
Great to hear. I'm just taking over a project at University of Oregon that uses Mukurtu, looking forward to testing out the v4 branch.

**Comment by mellydoll** [comment]
Dear folks,

On the Tribesourcing Southwest Film (https://tribesourcingfilm.org) site we use a workaround you provided us with in order to continue to use stacked media players instead of the carousel type. In order for our project to work properly we want to make sure we can continue to either make this slight edit (see your instructions below) to maintain the stacked structure, or perhaps you can make it optional for the user to choose which they like better?  Thank you! Melissa 

In cPanel, open File Manager
Depending on your folder structure, find ‘public_html’ and navigate the following folder structure: sites > all > themes > mukurtu > templates > fields
That last folder should have a single file called ‘field—field-media-asset.tpl.php’. Right click on the file and rename it (I suggest something like ‘field—field-media-asset.tpl.php_override’ so you can easily rename back to default if ever needed).
Then on your site, as a drupal administrator, hover over the home icon in the drupal toolbar and from the dropdown click “Flush all caches”.

**Reply by michael-wynne-wsu** [reply]
Thanks Melissa, I've actually started a new discussion for look and feel topics (we didn't have one active before) and I copied/referenced this over there: https://github.com/MukurtuCMS/Mukurtu-CMS/discussions/35 

We haven't started work on theming/look and feel so it will be good to get this included in a big list over there.

**Reply by mellydoll** [reply]
Thank you Michael!
